### PR TITLE
OCPBUGS-36901: Add step to network observability installation for Lokiless option

### PR DESCRIPTION
**Pull request by skrthomas**
…i-less option

Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OCPBUGS-36901
Link to docs preview:

Step 5 b: https://79278--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/installing-operators.html#network-observability-operator-installation_network_observability
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
Hey @memodi @jotak when reviewing Steven's what's new, I realized we didn't capture this step in the docs. Can you PTAL and lmk if its OK? I already pinged Joel about this on Slack when I realized and he said its ok.

**Review by memodi — APPROVED** [review]
/lgtm
thanks @skrthomas

File: modules/network-observability-operator-install.adoc [hunk 1]
```diff
@@ -33,6 +33,7 @@ Additionally, this installation example uses the `netobserv` namespace, which is
 . Navigate to *Operators* -> *Installed Operators*. Under Provided APIs for Network Observability, select the *Flow Collector* link.
 . Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
 .. *spec.agent.ebpf.Sampling*: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", `spec.agent.ebpf`.
+.. If you are not using Loki, click *Loki client settings* and change *Enable* to *False*. It is enabled by default.
```
Comment (aspauldi) [inline modules/network-observability-operator-install.adoc:36]:
```suggestion
.. If you are not using Loki, click *Loki client settings* and change *Enable* to *False*. The setting is enabled by default.
```

**Review by aspauldi — COMMENTED** [review]
Just one little thing regarding the use of "It" and clarifying what is being referred to. (I wasn't sure whether to suggest changing it to 'The setting' or 'This setting.')

Thanks!

**Comment by skrthomas** [comment]
@jotak I know we discussed over Slack already, but if this looks ok can you approve here too?

**Review by jotak — APPROVED** [review]
lgtm, thanks!

**Pull request merged**